src/services/refactors/extractSymbol.ts

```diff
@@ -1112,6 +1117,12 @@ namespace ts.refactor.extractSymbol {
         const renameLocation = getRenameLocation(edits, renameFilename, localNameText, /*isDeclaredBeforeUse*/ true);
         return { renameFilename, renameLocation, edits };
 
+        function isInJSXContent(node: Node): node is JsxOpeningLikeElement {

```

Comment:
Looks like this return type should be `node is JsxElement` if a type predicate is needed, but it doesn’t look like it’s needed.

```suggestion
        function isInJSXContent(node: Node) {
```